ros2run/ros2run/command/run.py

```diff
@@ -32,6 +32,12 @@ def add_arguments(self, parser, cli_name):
             help="Prefix command, which should go before the executable. "
                  "Command must be wrapped in quotes if it contains spaces "
                  "(e.g. --prefix 'gdb -ex run --args').")
+        try:
+            from argcomplete.completers import SuppressCompleter
+        except ImportError:
+            raise
```

Comment:
Maybe I'm missing something here, but if we raise here, how do "gracefully fall back"